src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs

```diff
@@ -53,8 +53,8 @@ public static bool IsSettingEnabled(string settingName)
         private static IConfigurationRoot BuildConfiguration()
         {
             var configurationBuilder = new ConfigurationBuilder()
-                .AddEnvironmentVariables()
-                .AddJsonFile("appsettings.json", optional: true);
+                .AddJsonFile("appsettings.json", optional: true)
+                .AddEnvironmentVariables();
```

Comment:
So now we're consistent with the v2.x behavior that was here: https://github.com/Azure/azure-webjobs-sdk/blob/v2.x/src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs#L21

Reply:
It's actually the opposite of the previous behavior. I mentioned that during triage, but with this changes, the default will differ than what we had in V2